test/serving.bash

```diff
@@ -128,14 +125,10 @@ function run_knative_serving_rolling_upgrade_tests {
   PROBER_PID=$!
 
   if [[ $UPGRADE_SERVERLESS == true ]]; then
-    prev_serving_version=$(oc get knativeserving.operator.knative.dev knative-serving -n $SERVING_NAMESPACE -o=jsonpath="{.status.version}")
-    # This is ugly hack. Use KNATIVE_SERVING_VERSION if issues/361 was solved.
-    latest_serving_version=$(sed -n 's/^.*Version.*"\(.*\)".*$/\1/p' ${rootdir}/knative-operator/vendor/knative.dev/serving-operator/version/version.go)
```

Comment:
latest_serving_version needs to be defined in the following way here:
```
latest_serving_version="${KNATIVE_SERVING_VERSION/v/}"
```
The upgrade tests currently fail because we use it a few lines below and it's undefined.